playbooks/viya-mmsu/README.md

```diff
@@ -51,6 +51,14 @@ To start the minimally required set of services prior to starting a Restore, exe
 ```
 ansible-playbook viya-ark/playbooks/viya-mmsu/viya-services-restore.yml
 ```
+To disable sas-viya-all-services to run on system reboot, execute:
+```
+ansible-playbook viya-ark/playbooks/viya-mmsu/viya-services-disable.yml
+```
```

Comment:
this playbook is already referenced in line 34 above, perhaps remove that section to put these together here at the end instead

Reply:
I didn't completely remove reference to the disable playbook, but removed descriptive text.

Reply:
It's duplicated information about the `viya-services-disable.yml` playbook

https://github.com/sassoftware/viya-ark/blob/2146460b183ac6a6455685217a328c25c9ebd337/playbooks/viya-mmsu/README.md?plain=1#L34-L37

Reply:
ok.  i'll completely remove it from the top.  i felt it left the top (requirements section) with an instruction to do something but now without the ability to know exactly what needed to do.